main.tf

```diff
@@ -2,12 +2,15 @@
 terraform {
   required_providers {
     azurerm = {
-      source  = "hashicorp/azurerm"
-      version = "~>1.31"
+      source = "hashicorp/azurerm"
```

Comment:
Do we want to tag a version here? so the config works even when v3 is released?

Reply:
Docs recommend that users always constrain provider versions, so yeah, we should probably try to set a good example. 😅